infoTree/getURLfile.js

```diff
@@ -0,0 +1,18 @@
+function getURLindex(label){
+    //first find which URL maps to it
+    let targetURL = "";
+    for (int i = 0; i < whereDoesItGo.length; ++i){
+        let cur = whereDoesItGo[i];
+        if (cur.label === label)
+            targetURL = cur.value;
+    }
```

Comment:
this first loop could be replaced with 
`const targetURL= whereDoesItGo.find(el => el.label === label)`

Reply:
Hugely in favor of this change, pushing update